### ISSUE TITLE
add BC7 texture compression

### ISSUE THREAD

**Issue by ManoaNosea** [dhewm/dhewm3#246]
hi guys, I am working on a (complete) texture mod for this game and DXT is many many years old in term of quality and ratio, I ask if you could add BC7 to the game

**Comment by turol** [comment]
This might cause issues on older GPUs or other platforms which don't support BC7. I think Mac OpenGL doesn't, not sure. If we add new texture format it should be [basis](https://github.com/BinomialLLC/basis_universal).

**Comment by ManoaNosea** [comment]
looks nice :)

**Comment by DanielGibson** [comment]
Yeah, basis would make more sense than BC7, but no idea how much work that'd be

**Comment by ManoaNosea** [comment]
trust me I will make it worth :)

**Comment by RinMaru** [comment]
> trust me I will make it worth :)
> 
> http://manoa.dream.org.il/Pics/doom3/a_lfwall16a-new.png
> http://manoa.dream.org.il/Pics/doom3/delel-post2.png
> http://manoa.dream.org.il/Pics/doom3/elevator_door.png
> http://manoa.dream.org.il/Pics/doom3/hatch.png
> http://manoa.dream.org.il/Pics/doom3/shot00005_proc.png
> http://manoa.dream.org.il/Pics/doom3/shot00131_proc.png
> http://manoa.dream.org.il/Pics/doom3/shot00132_proc.png
> http://manoa.dream.org.il/Pics/doom3/shot00133_proc.png
> http://manoa.dream.org.il/Pics/doom3/shot00178-specular-ENET_proc.png
> 
> and more manys and goods :)

Be a nice place to put Esrgan to the test
